src/sagas/development.saga.test.js

```diff
@@ -16,14 +16,12 @@ describe('development saga', () => {
 
   describe('Reset state', () => {
     it('should reset state to initialState', () => {
-      const electronStore = new ElectronStore();
       const saga = handleShowResetDialog();
 
       // Show dialog
       saga.next();
 
       // Confirm & check that electronStore.clear is called
-      // Note: No need to check that the config.json is reset as this is tested in the library - just test that it is called

```

Comment:
I removed this comment because I actually found it kinda confusing (I'm guessing `config.json` is what `electron-store` creates internally?).

And yeah I feel like once you get used to saga tests, it's unsurprising that it's not checking that library functions are executed, just that the saga is calling the right functions.

Reply:
Yeah, it's storing the state there (the path to that file is inside of `saga.next().value`). I've looked at that file during first manual tests to see that the clearing is working.

It's OK to remove that comment.